templates/solutions/open-source-security/cyber-resilience-act/index.html

```diff
@@ -0,0 +1,591 @@
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block outer_content %}
+
+  {% block content %}
+
+    <section class="p-section--hero">
+      <div class="row--50-50-on-large">
+        <div class="col">
+          <h1 class="u-no-margin--bottom">Meet your Cyber Resilience Act requirements with Canonical</h1>
+        </div>
+        <div class="col">
+          <div class="p-section--shallow u-hide--large"></div>
+          <div class="p-section--shallow">
+            <p>
+              Build your products and device software with Ubuntu and Canonical's trusted open source portfolio. Fast-track your pathway to CRA compliance.
+            </p>
+          </div>
+          <div class="p-cta-block">
+            <a href="#"
+               aria-controls="contact-modal"
+               class="p-button--positive js-invoke-modal">Contact our team</a>
+          </div>
+        </div>
+      </div>
+    </section>
+    <hr class="p-rule is-fixed-width" />
+
+    <div class="u-fixed-width">
+      <section class="p-section">
+        <div class="row--50-50">
+          <div class="col">
+            <h2>
+              What is the
+              <br />
+              Cyber Resilience Act (CRA)?
+            </h2>
+          </div>
+          <div class="col">
+            <p class="u-embedded-media">
+              <iframe title="What is the Cyber Resilience Act?"
+                      class="u-embedded-media__element"
+                      src="https://www.youtube.com/embed/ltBtIDvav6c"
+                      allowfullscreen=""
+                      frameborder="0"></iframe>
+            </p>
+            <p class="p-section--shallow u-no-padding--bottom">
+              The CRA is a European Union legislation that aims to make Products with Digital Elements (PDEs) safer by requiring developers, manufacturers, distributors and retailers to follow mandatory cybersecurity, documentation, and vulnerability reporting requirements. The CRA extends this protection throughout the product life cycle.
+            </p>
+            <hr class="p-rule is-muted" />
+            <a href="https://ubuntu.com/blog/a-cisos-comprehensive-breakdown-of-the-cyber-resilience-act">
+              Read our CISO's full breakdown of the CRA ›
+            </a>
+          </div>
+        </div>
+      </section>
+    </div>
+
+    <hr class="p-rule is-fixed-width" />
+
+    <div class="u-fixed-width">
+      <section class="p-section">
+        <div class="row--50-50">
+          <div class="col">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/c46635ff-what-products-and-devices.png",
+            alt="",
+            width="1200",
+            height="1801",
+            hi_def=True,
+            loading="auto|lazy"
+            ) | safe
+            }}
```

Comment:
Can you format the image template pls
```suggestion
            {{ image(url="https://assets.ubuntu.com/v1/c46635ff-what-products-and-devices.png",
                            alt="",
                            width="1200",
                            height="1801",
                            hi_def=True,
                            loading="lazy") | safe
            }}
```